src/Microsoft.AspNet.Mvc.Core/CacheProfile.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Mvc
+{
+	public class CacheProfile
+	{
+        public CacheProfile(string name)
+        {
+            Name = name;
+        }
+
+        /// <summary>
+        /// Gets or sets the name of the CacheProfile.
+        /// The CacheProfile is referred in the ResponseCache attribute using this name.
+        /// This is a required parameter.
+        /// </summary>
+        public string Name { get; set; }
+
+		/// <summary>
+        /// Gets or sets the duration in seconds for which the response is cached.
+        /// This sets "max-age" in "Cache-control" header.
+        /// </summary>
+        public int? Duration { get; set; }
+
+        /// <summary>
+        /// Gets or sets the location where the data from a particular URL must be cached.
+        /// This sets the value for "Cache-control" header.
```

Comment:
probably this line should read more like ```If this property is set to a non null value,  the "cache-control" header is set in the <nit: <see cref="HttpContext.Response" />```  and should go in as remarks.
Similar comments for others